resource/cache/client.lua

```diff
@@ -51,6 +52,10 @@ CreateThread(function()
 
 		cache:set('weapon', hasWeapon and currentWeapon or false)
 
+		local playername = GetPlayerName(PlayerId())
+
+		cache:set('playername', playername)
```

Comment:
here also use camelCase

Reply:
I pushed another commit. should be all good now, thanks for the heads up